test/builtin_dataset_mocks.py

```diff
@@ -1431,3 +1431,19 @@ def stanford_cars(info, root, config):
         make_tar(root, "car_devkit.tgz", devkit, compression="gz")
 
     return num_samples
+
+
+@register_mock
+def sun397(info, root, config):
+    num_samples = 2
+    categories = ["abbey", "bakery/shop", "bar"]
+    for category in categories:
+        create_image_folder(
+            root=root,
+            name="{}/{}".format(category[0], category),
```

Comment:
This will break on Windows. If the `/` actually means subdirectory, we need to handle that.

Reply:
Got it, I think I can fix this with `pathlib`. Thanks!